File: LeetCode/Easy/1323-maximum-69-number/1323-maximum-69-number.java
class Solution {
    public int maximum69Number (int num) {
        int ret = num;
        int curr = 0;
        int digit = -1;

        while (num > 0) {
            if (num % 10 == 6) 
                digit = curr;
            
            num /= 10;
            curr++;
        }

        if (digit > -1) ret += 3 * Math.pow(10, digit);

        return ret;
    }
}

File: LeetCode/Easy/1323-maximum-69-number/1323-maximum-69-number.py
class Solution:
    def maximum69Number (self, num: int) -> int:
        ret = num
        curr = 0
        digit = -1

        while (num > 0):
            if num % 10 == 6:
                digit = curr
            
            num //= 10
            curr += 1
        

        if digit > -1:
            ret += 3 * 10 ** digit

        return ret